templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
If the components are zipped up, you can use `Get-ChocolateyUnzip` if it is local to the folder (embedded in the package).

~~~
$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
$gzipFile = Join-Path $toolsDir 'somefile.tar.gz'
$destination = 'C:\tools\pl-build-tools'

# locally embedded
Get-ChocolateyUnzip $gzipFile $destination -packageName 'test'

# if it is remote, you can just use the following pattern
Install-ChocolateyZipPackage $packageName $url $toolsDir $url64
~~~